### ISSUE TITLE
ImportWarning triggered by import shapely.geometry

### ISSUE THREAD

**Issue by pope1ni** [Toblerity/Shapely#774]
## Expected behavior and actual behavior.

An `ImportWarning` is triggered, it shouldn't be.

## Steps to reproduce the problem.

```
$ python -We -c 'import shapely.geometry'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/pope1ni/.virtualenvs/polaris-py3/lib/python3.7/site-packages/shapely/geometry/__init__.py", line 23, in <module>
    import shapely.speedups
  File "/home/pope1ni/.virtualenvs/polaris-py3/lib/python3.7/site-packages/shapely/speedups/__init__.py", line 11, in <module>
    from shapely.speedups import _speedups
  File "shapely/speedups/_speedups.pyx", line 8, in init shapely.speedups._speedups
ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
```

## Operating system

Arch Linux

```
Linux localhost 5.2.2-arch1-1-ARCH #1 SMP PREEMPT Sun Jul 21 19:18:34 UTC 2019 x86_64 GNU/Linux
```

## Shapely version and provenance

1.6.4.post2 installed from PyPI using pip

**Comment by yajinwuzl** [comment]
Is this problem solved?I am experiencing the same problem, my problem is that "importerror" cannot be from shapely.gemetry import Point

**Comment by sgillies** [comment]
The problem was in the _speedups.c code generated by Cython. Other packages were affected, too: https://github.com/astropy/astropy/issues/6025. It seems to have been resolved between Cython versions 0.19 (used to make the 1.6.4 wheels) and 0.29.5 (used to make the 1.7a2 wheels) because I don't see any sign of it with the 1.7a2 pre-release. `pip install -U --pre shapely` if you want to try the pre-release.

I'm going to close this one and mark it wontfix.

**Comment by yajinwuzl** [comment]
The issue already got of me, thanks!
